### PR TITLE
Create GCSO

### PR DESCRIPTION
**Pull request by Hari328384**

### CODE REVIEW

**Comment by Hari328384** [comment]
please

**Comment by KrishGarg** [comment]
> please

bruh why are you making a pull request of an empty file?

**Comment by KrishGarg** [comment]
By the way, you have 24 hours to reply to me otherwise, I will be closing this pull request.

File: GCSO [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (seema3344) [inline GCSO:1]: q

Comment (seema3344) [reply GCSO:1]: haeta5

**Comment by KrishGarg** [comment]
I don't think i really want to accept the request.

**Pull request closed without merging**